ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -56,6 +60,9 @@
     mClient = client;
     mShuttingDown = false;
     mDefaultUserAgent = defaultUserAgent;
+    mCookieManager = new CookieManager();
+    mCookieManager.setCookiePolicy(CookiePolicy.ACCEPT_ORIGINAL_SERVER);
```

Comment:
This is the default (https://docs.oracle.com/javase/7/docs/api/java/net/CookieManager.html#setCookiePolicy(java.net.CookiePolicy)) but doesn't hurt to be explicit here